src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,11 +5,27 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private double loadCapacity;
```

Comment:
Consider using access modifiers for the fields `loadCapacity` and `cargoVolume`. It's a good practice to encapsulate the fields of a class and provide public getter and setter methods to access them.